docs/development/survey/initial_summary.md

```diff
@@ -0,0 +1,79 @@
+
+Out of users within Substrate ecosystem, 3/15 responded that they do not use derivations, and 3/15 refused to tell. One user in that group did not know what derivations are.
+
+No significant corellation was observed between user role in ecosystem and use of derivations. This is certainly due to lack of data, but it is also quite clear that there are 1-address developer users and multiadress stakeholders.
+
+## Suggestions
+
+Users have noted poor usability of hardware wallets.
+
+As always in similar discussion, hassle of metadata updates and lack of available manuals was noted. Demand for simpler version of everything was observed.
+
+Ledger's low rate of updates was mentioned as unacceptable.
+
+Users want support of Shamir secret sharing integrated.
+
+Users care about whole system being opensource.
+
+Users mentioned that they value mechanical robustness and quality of build.
+
+One of key demands for key managers noted from the feedback is deep integration into existing wallets. Apparently many features of the networks are slowly becoming available in wallets as development proceeds, placing any additional barriers in their wake like keeping a separate client app up to date would only reduce usability of Kampela to minimal and introduce whole lot of new threats caused by extra code maintenance.
+
+# Conclusions
+
+1. We need to keep collecting information. Fortunately, the project is organized in a way that we could easily keep collecting data and reevaluating analysis till the end, relying on the latest information to plan the next stage.
+
+2. H1 seems to be confirmed. There is still hope that users would become more open and willing to cooperate as project moves through milestones 3 and 4 delivering more tangible and visualizeable results and we will suddenly have more high quality information.
+
+3. 4 competing well-known systems were identified. We should learn from those systems, try to adapt their strong sides and fix their shortcomings.
+
+4. We should limit the need to update devices. Ideally they should be usable at all times and in case of global breach just be recalled or replaced at users' expence.
+
+5. We should focus on standard communication protocol and integration into existing wallets (at least Nova who actively collaborates and helped us gather data for this report). We could also synergize this project into our proposal in making for a functionality-centered wallet project Kiondo.
+
+6. We should remember to include banana split protocol on-board
+
+7. Users do not seem to differentiate by roles in ecosystem, but rather by their technical skill and threat models. We should reconsider user types classification. Fortunately, we have a list of user desires, and we can collect more data on this later, since we have a couple of milestones ahead before complete prototype design.
```

Comment:
```suggestion
7. Users do not seem to differentiate by roles in ecosystem, but rather by their technical skill and expected attack routes. We should reconsider user types classification. Fortunately, we have a list of user desires, and we can collect more data on this later, since we have a couple of milestones ahead before complete prototype design.
```